### PR TITLE
Added action to initiate the upgrade server process

### PR DESCRIPTION
**Pull request by jsuchome**
This is preliminary location. In needs to be bound to the new upgrade API...

Depends on https://github.com/crowbar/crowbar/pull/2258

### CODE REVIEW

File: crowbar_framework/app/controllers/installer/upgrades_controller.rb [hunk 1]
```diff
@@ -91,6 +91,29 @@ def start
       end
     end
 
+    # This will initiate the upgrade of admin server.
+    # Once this function is called, server packages will be upgraded to latest versions.
+    # Right after all packages are upgraded, server will be rebooted.
```
Comment (jsuchome) [inline crowbar_framework/app/controllers/installer/upgrades_controller.rb]: Honestly, I do not see what's wrong here.

**Comment by MaximilianMeister** [comment]
@jsuchome the new API is now available, just add it to:
https://github.com/crowbar/crowbar-core/blob/master/crowbar_framework/app/controllers/api/crowbar_controller.rb#L35

**Comment by jsuchome** [comment]
@MaximilianMeister as mentioned on #irc, it needs the API in 3.0 branch

**Comment by rsalevsky** [comment]
@jsuchome Not sure, but makes it sense to have it first in master and then create a backport? Otherwise we will have for the C8 upgrade the same problem that we no code at all.

**Comment by MaximilianMeister** [comment]
@jsuchome I moved the code to https://github.com/crowbar/crowbar-core/issues/604 
thanks for the initial work

**Pull request closed without merging**